icons/shield_us_nm_frontage.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 5.292 5.292" xmlns="http://www.w3.org/2000/svg">
+ <path style="fill:#fff;fill-opacity:1;stroke:#000;stroke-width:.26458333;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" d="M 5.1593749,0.13229166 H 0.13229166 V 5.1593749 l 0.79375001,10e-8 V 4.8947917 H 2.2489583 L 2.1166667,4.6302083 h 3.0427082 z"/>
```

Comment:
Are these shields supposed to be black on white, or white on brown as in this photo?

[![](https://upload.wikimedia.org/wikipedia/commons/thumb/7/7a/NM1035_Frontage_Road_Sign_%2829963269246%29.jpg/640px-NM1035_Frontage_Road_Sign_%2829963269246%29.jpg)](https://commons.wikimedia.org/wiki/File:NM1035_Frontage_Road_Sign_%2829963269246%29.jpg)

Reply:
Perhaps signage is inconsistent, but I was able to find more black-on-white examples from more recent imagery.
* https://www.bing.com/maps?toWww=1&redig=5E1C7ABAF707476588549A54993AF738&cp=32.173542%7E-106.671324&lvl=21.2&style=x&mo=om.1
* https://www.bing.com/maps?toWww=1&redig=5E1C7ABAF707476588549A54993AF738&cp=32.2661%7E-106.738429&lvl=21.6&mo=om.1&style=x
* https://www.bing.com/maps?toWww=1&redig=5E1C7ABAF707476588549A54993AF738&cp=32.284562%7E-106.879612&lvl=20.1&mo=om.1&style=x

Reply:
Commons as [a couple photos](https://commons.wikimedia.org/wiki/Category:Frontage_roads_in_New_Mexico) of these brown frontage road signs, including one on the state’s longest frontage road.

[This page](https://www.aaroads.com/shields/show.php?image=NM19861281) claims the black on white version is from a 1986 specification. [A 2005 specification](https://web.archive.org/web/20060830021354if_/http://nmshtd.state.nm.us:80/upload/images/Access_Management/NMDOT-2005-Sign-Code-Listing.pdf#page=118) looks similar. However, neither the signs you found nor the ones on Commons conforms to this specification. The NMDOT Signing and Striping Manual was updated in 2008, but I’ve been unable to find a newer edition of the sign code listing since 2005.

Reply:
Let's introduce a variable that randomly chooses between the two.

Reply:
The white-on-brown FR&nbsp;1035 sign pictured above in 2014 stood until at least [2015](https://www.google.com/maps/@32.0242747,-106.5912269,3a,15y,18.54h,87.47t/data=!3m7!1e1!3m5!1slXF46v-KmbbT-1GslAx7Cg!2e0!5s20150701T000000!7i13312!8i6656), but by [2018](https://www.google.com/maps/@32.0242747,-106.5912269,3a,15y,353.59h,87.68t/data=!3m7!1e1!3m5!1slXF46v-KmbbT-1GslAx7Cg!2e0!5s20150701T000000!7i13312!8i6656) it had been replaced by the black-on-white sign. The other white-on-brown sign on Commons, for FR&nbsp;1043, [remains](https://www.google.com/maps/@32.0242747,-106.5912269,3a,15y,353.59h,87.68t/data=!3m7!1e1!3m5!1slXF46v-KmbbT-1GslAx7Cg!2e0!5s20150701T000000!7i13312!8i6656), but [a reassurance marker](https://www.openstreetmap.org/node/10078036144) for that route is black on white. I think this is pretty conclusive that the brown sign was an aberration or an older design.